### PR TITLE
works for "current_version" 

### PR DESCRIPTION
**Pull request by ndushay**
tests pass, all sorts of goodness.  Basically, ready for releasing gem version 0.1.0.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Layout/EmptyLinesAroundClassBody:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - 'spec/**/*'
+
+Metrics/LineLength:
+  Exclude:
+    - 'spec/**/*'
```
Comment (jcoyne) [inline .rubocop.yml]: I'm not keen on unlimited line length, even for specs.  I'd prefer we set a sensible `Max` as the default is annoyingly low

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+# Offense count: 1
+Metrics/AbcSize:
+  Max: 23
+
+# Offense count: 1
+# Configuration parameters: CountComments, ExcludedMethods.
+# ExcludedMethods: refine
+Metrics/BlockLength:
+  Max: 26
```
Comment (jcoyne) [inline .rubocop_todo.yml]: I'd suggest adding a BlockLength exception for spec/**/* in .rubocop.yml

Comment (ndushay) [reply .rubocop_todo.yml]: it's there.  :-)

File: .rubocop.yml (hunk 1 above)
Comment (ndushay) [reply .rubocop.yml]: done.

**Comment by ndushay** [comment]
I'm gonna merge this with one approval;  want to hook it up and shake out probs asap.

I adjusted rubocop per jcoyne's suggestions.

**Pull request merged**